src/olympia/devhub/tests/test_forms.py

```diff
@@ -762,3 +763,11 @@ def test_error_invalid_user(self):
         self.form = EditThemeOwnerForm({'owner': 'omg@org.yes'},
                                        instance=self.instance)
         assert not self.form.is_valid()
+
+
+class TestDistributionChoiceForm(TestCase):
+
+    def test_lazy_choice_labels(self):
+        """Tests that the labels in `choices` are still lazy"""
+        fields = forms.DistributionChoiceForm.base_fields
+        assert isinstance(fields['choices'].choices[0][1], Promise)
```

Comment:
Yeah, it's not immediately clear what this assertion is doing. I would suggest a comment like what you said or maybe: 'This makes sure that the value is a proxy object instead of the real value'